notices.json

```
@@ -1,4 +1,20 @@
 [
+  {
+    "id": "boards_deprecations",
+    "conditions": {
+      "audience": "sysadmin",
+      "instanceType": "cloud",
```

```
cat notices.json | jq                                                                                                                                      
parse error: Expected another key-value pair at line 7, column 5
```


```suggestion
      "instanceType": "cloud"
```
This should produce a valid json

---

Build is failing because `ajv validate`( JSON Schema Validator ) which is part of the CI process,  is failing .